lib/scrivener/html.ex

```diff
@@ -152,13 +152,19 @@ defmodule Scrivener.HTML do
           if paginator.page_number == page_number do
             classes = ["active"]
           end
-          params_with_page = Keyword.merge(url_params, [{page_param, page_number}])
-          content_tag :li, class: Enum.join(classes, " ") do
-            to = apply(path, args ++ [params_with_page])
-            if to do
-              link "#{text}", to: to
-            else
-              content_tag :a, "#{text}"
+          if text == :ellipsis do
+            content_tag(:li) do
+              [content_tag(:span, "&hellip;")]
```

Comment:
I wonder if we can instead of having ellipsis be `true` / `false`, have ellipsis default to `"&hellip;"` that way the can render whatever they want in place of it if need be.